cmd/repo-updater/repos/store.go

```diff
@@ -834,6 +1153,20 @@ func metadataColumn(metadata interface{}) (msg json.RawMessage, err error) {
 	return
 }
 
+func sourcesColumn(repoID api.RepoID, sources map[string]*SourceInfo) (msg json.RawMessage, err error) {
```

Comment:
```suggestion
func sourcesColumn(repoID api.RepoID, sources map[string]*SourceInfo) (json.RawMessage, error) {
```

Besides, can we have docstring for this function?